app/views/sessions/verify.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render 'shared/page_title', title: "Account Verified" %>
+
+<div class="row">
+  <%= hidden_field_tag "account_verified", @account_verified  %>
```

Comment:
why do you need this line here?

Reply:
you're correct, we don't need. We already work it in the controller